ui/Screen/Profile/Tracking.svelte

```diff
@@ -34,9 +34,27 @@
   ];
 
   const untracked = [
-    { urn: "hwd1yreg4khbjfa4gsyrio3f7ehluwkdhyregs4k" },
-    { urn: "fjkldasjfkdlsajfio943we859043ikjioclesdjf" },
-    { urn: "fjkldasjfkdlsajfio39409340390we859043ikjioclesdjf" },
+    {
+      urn: "hwd1yreg4khbjfa4gsyrio3f7ehluwkdhyregs4k",
+      metadata: {
+        name: "snickers",
+        description: "project description",
```

Comment:
We won't have any other metadata available, aside from the name (which we only would get if it's included in the urn).

```suggestion
```

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/906/commits/817faffb393cc5d50d70605e357d87a6cb7dc544